pkg/controllers/management/auth/prtb_handler.go

```diff
@@ -24,21 +24,12 @@ const (
 )
 
 var projectManagementPlaneResources = map[string]string{
-	"apps":                        "project.cattle.io",
-	"apprevisions":                "project.cattle.io",
-	"catalogtemplates":            "management.cattle.io",
-	"catalogtemplateversions":     "management.cattle.io",
 	"sourcecodeproviderconfigs":   "project.cattle.io",
-	"projectcatalogs":             "management.cattle.io",
-	"projectmonitorgraphs":        "management.cattle.io",
+	"projectloggings":             "management.cattle.io",
```

Comment:
Could you remove `projectloggings`. My PR removing it https://github.com/rancher/rancher/pull/47997